tests/perf-test/ReportActionsList.perf-test.js

```diff
@@ -0,0 +1,188 @@
+    }),
+);
+
+afterAll(() => {
+    jest.clearAllMocks();
+});
+
+const mockOnLayout = jest.fn();
+const mockOnScroll = jest.fn();
+const mockLoadMoreChats = jest.fn();
+const mockRef = {current: null};
+
+// Initialize the network key for OfflineWithFeedback
+beforeEach(() => {
+    PusherHelper.setup();
+    wrapOnyxWithWaitForBatchedUpdates(Onyx);
+    return Onyx.merge(ONYXKEYS.NETWORK, {isOffline: false});
+});
+
+// Clear out Onyx after each test so that each test starts with a clean slate
+afterEach(() => {
+    Onyx.clear();
+    PusherHelper.teardown();
+});
+
+const getFakeReportAction = (index) => ({
+    actionName: 'ADDCOMMENT',
+    actorAccountID: index,
+    automatic: false,
+    avatar: '',
+    created: '2023-09-12 16:27:35.124',
+    isAttachment: true,
+    isFirstItem: false,
+    lastModified: '2021-07-14T15:00:00Z',
+    message: [
+        {
+            html: 'hey',
+            isDelatedParentAction: false,
+            isEdited: false,
+            reactions: [],
+            text: 'test',
+            type: 'TEXT',
+            whisperedTo: [],
+        },
+    ],
+    originalMessage: {
+        html: 'hey',
+        lastModified: '2021-07-14T15:00:00Z',
+    },
+    pendingAction: null,
+    person: [
+        {
+            type: 'TEXT',
+            style: 'strong',
+            text: 'email@test.com',
+        },
+    ],
+    previousReportActionID: '1',
+    reportActionID: index.toString(),
+    reportActionTimestamp: 1696243169753,
+    sequenceNumber: 2,
+    shouldShow: true,
+    timestamp: 1696243169,
+    whisperedToAccountIDs: [],
+});
+
+const getMockedSortedReportActions = (length = 100) => Array.from({length}, (__, i) => getFakeReportAction(i));
+
+const currentUserAccountID = 5;
+
+function ReportActionsListWrapper() {
+    return (
+        <ComposeProviders components={[OnyxProvider, LocaleContextProvider, WindowDimensionsProvider, ReportAttachmentsProvider]}>
+            <ReactionListContext.Provider value={mockRef}>
+                <ActionListContext.Provider value={mockRef}>
+                    <ReportActionsList
+                        sortedReportActions={getMockedSortedReportActions(500)}
+                        report={LHNTestUtils.getFakeReport()}
+                        onLayout={mockOnLayout}
+                        onScroll={mockOnScroll}
+                        loadMoreChats={mockLoadMoreChats}
+                        currentUserPersonalDetails={LHNTestUtils.fakePersonalDetails[currentUserAccountID]}
+                    />
+                </ActionListContext.Provider>
+            </ReactionListContext.Provider>
+        </ComposeProviders>
+    );
+}
+
+test('should render ReportActionsList with 500 reportActions stored', () => {
+    const scenario = async () => {
+        await screen.findByTestId('report-actions-list');
+        const hintText = Localize.translateLocal('accessibilityHints.chatMessage');
+        // Esure that the list of items are rendered
```

Comment:
```suggestion
        // Ensure that the list of items is rendered
```

Reply:
Good catch, fixed. 

These are the results after adding scenarios to the `ReportActionsList` and increasing the number of rendered reports from 10 to 100. Here is an example of the updated result for 500 reports:

<img width="1023" alt="Screenshot 2023-10-18 at 10 27 04" src="https://github.com/Expensify/App/assets/19835085/16f700eb-3182-49e0-8ab6-c28014c39646">